tests/unit/test_federated_s3.py

```diff
@@ -48,6 +48,11 @@ async def test_get_configs_by_alias(federated_s3: FederatedS3Fixture):
 
 async def test_populate_dummy_items(federated_s3: FederatedS3Fixture):
     """Test the populate_dummy_items function on the FederatedS3Fixture."""
+    # Explicitly calling this again, as it seems to solve the Github Actions issue
+    # Needs more investigation on what might be happening during fixture setup that makes this necessary
```

Comment:
I noticed this as well, happens very rarely. Maybe the bucket deletion takes some time?